src/qubex/experiment/mixin/characterization_mixin.py

```
@@ -1122,7 +1147,7 @@ def obtain_coupling_strength(
         target_qubit: str,
         spectator_qubit: str,
         *,
-        time_range: ArrayLike = np.arange(0, 5001, 200),
+        time_range: ArrayLike = np.arange(0, 20001, 500),
```

Ruff (0.8.2)

Do not perform function call `np.arange` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)